src/utils/decorators/input-validation/__spec__.js

```diff
@@ -189,13 +189,15 @@ describe('InputValidation', () => {
 
       describe('when the next value does not match the current value', () => {
         it('does not call validate if it is the currently active input', () => {
-          let wrapper = shallow(<Component />);
-          instance = wrapper.instance();
+          let component = <Component />;
+          let wrapper = shallow(component);
+          instance = TestUtils.renderIntoDocument(component);
```

Comment:
this seems like a bit of a workaround - do we need to mount the component in this case?